injected/src/content-feature.js

```diff
@@ -144,121 +137,11 @@ export default class ContentFeature {
-        }
-        const domainMatch = [...this.matchDomainFeatureSetting('domains')].sort((a, b) => {
-            return a.domain.length - b.domain.length;
-        });
-        for (const match of domainMatch) {
-            if (match.patchSettings === undefined) {
-                continue;
-            }
-            try {
-                result = immutableJSONPatch(result, match.patchSettings);
-            } catch (e) {
-                console.error('Error applying patch settings', e);
-            }
-        }
-        return result?.[featureKeyName];
-    }
-
-    /**
-     * Return the settings object for a feature
-     * @param {string} [featureName] - The name of the feature to get the settings for; defaults to the name of the feature
-     * @returns {any}
-     */
-    _getFeatureSettings(featureName) {
-        const camelFeatureName = featureName || camelcase(this.name);
-        return this.#args?.featureSettings?.[camelFeatureName];
-    }
-
-    /**
-     * For simple boolean settings, return true if the setting is 'enabled'
-     * For objects, verify the 'state' field is 'enabled'.
-     * This allows for future forwards compatibility with more complex settings if required.
-     * For example:
-     * ```json
-     * {
-     *    "toggle": "enabled"
-     * }
-     * ```
-     * Could become later (without breaking changes):
-     * ```json
-     * {
-     *   "toggle": {
-     *       "state": "enabled",
-     *       "someOtherKey": 1
-     *   }
-     * }
-     * ```
-     * This also supports domain overrides as per `getFeatureSetting`.
-     * @param {string} featureKeyName
-     * @param {string} [featureName]
-     * @returns {boolean}
-     */
-    getFeatureSettingEnabled(featureKeyName, featureName) {
-        const result = this.getFeatureSetting(featureKeyName, featureName);
-        if (typeof result === 'object') {
-            return result.state === 'enabled';
-        }
-        return result === 'enabled';
-    }
-
-    /**
-     * Given a config key, interpret the value as a list of domain overrides, and return the elements that match the current page
-     * Consider using patchSettings instead as per `getFeatureSetting`.
-     * @param {string} featureKeyName
-     * @return {any[]}
-     * @private
-     */
-    matchDomainFeatureSetting(featureKeyName) {
-        const domain = this.#args?.site.domain;
-        if (!domain) return [];
-        const domains = this._getFeatureSettings()?.[featureKeyName] || [];
-        return domains.filter((rule) => {
-            if (Array.isArray(rule.domain)) {
-                return rule.domain.some((domainRule) => {
-                    return matchHostname(domain, domainRule);
-                });
-            }
-            return matchHostname(domain, rule.domain);
-        });
-    }
-
     init(args) {}
 
     callInit(args) {
         const mark = this.monitor.mark(this.name + 'CallInit');
-        this.#args = args;
+        this.args = args;
```

Comment:
This becomes a method called `setArgs`. That we call from the constructor and `callInit`. We remove these two lines from `callLoad`.